curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/authentication-with-socket.io.md

```diff
@@ -33,7 +33,10 @@ io.use(
 );
 ```
 
-Be sure to add the `key` and `store` to the `session` middleware mounted on the app. This is necessary to tell *SocketIO* which session to relate to.
+You could note that configuring passport authentication for *Socket.IO* is very similar to configuring `session` middleware for our API. This is true because they are meant to use same authentication method - get session id from cookie and validate it.
+
+Beforehand for configuring `session` middleware we didn't set cookie name for session (`key`) explicitly - `express-session` package was using default value.
```

Comment:
```suggestion
Before when we configured the `session` middleware we didn't set the cookie name for session (`key`) explicitly. This is because the `session` package was using the default value.
```